src/BinaryKits.Zpl.Viewer/ZplAnalyzer.cs

```diff
@@ -63,16 +71,18 @@ public AnalyzeInfo Analyze(string zplData)
             {
                 var currentCommand = zplCommands[i];
```

Comment:
can we move the Trim to the top?

Reply:
hello !

I hesitated.

Indeed think we may share this, but I'm not 100% confident there is no risk for analyzer.Analyze(currentCommand) : 
I don't know how to ensure there will not be any side-effect :(

I propose a trimmedCommand only for start/end commands if you agree ?